### PR TITLE
Add age parameters

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
Adds the child and senior age parameters.

### CODE REVIEW

File: client/src/js/pages/policy/default_policy.js [hunk 1]
```diff
@@ -147,6 +147,14 @@ const DEFAULT_POLICY = {
...
+		title: "Adult UBI age threshold",
+		description: "Age at which a person receives the working-age adult UBI payment.",
+		default: 18,
+		value: 18,
+		max: 80,
+		summary: "Set the working-age UBI age threshold to @",
+	},
 	senior_UBI: {
 		title: "Senior UBI",
 		description: "Flat payment given to each person at or above State Pension age.",
```
Comment (MaxGhenis) [inline client/src/js/pages/policy/default_policy.js:160]: Should change

**Review by MaxGhenis — COMMENTED** [review]
Is it more common to define UBIs by age than State Pension age?

If so and we want to go this direction, I think there's also a FAQ question that needs to be updated.

**Comment by MaxGhenis** [comment]
Another consideration is having different definitions of working age and pensioner in the UBI vs the poverty breakout chart

**Comment by nikhilwoodruff** [comment]
@MaxGhenis yeah this has convinced me it's better to just leave the State Pension UBI as is - we'd have to adjust a lot of other parameters, and we don't want e.g. incorrect overlap with the State Pension and UBI plans.

**Pull request merged**